main.go

```
@@ -511,17 +511,90 @@ func isDueTodayFilter(g Goal) bool {
+// time-based tests.
 func isDueTomorrowFilterAt(g Goal, now time.Time) bool {
-	return IsDueTomorrowAt(g.Losedate, now) && !IsEndValueReached(g)
+	if IsEndValueReached(g) {
+		return false
+	}
+	return IsDueTodayAt(g.Losedate, now) || IsDueTomorrowAt(g.Losedate, now)
 }
 
-// isDueTomorrowFilter returns true if the goal is due tomorrow and hasn't already reached its end value
+// isDueTomorrowFilter returns true if the goal is due by the end of tomorrow and hasn't already reached its end value
 func isDueTomorrowFilter(g Goal) bool {
```

golangci-lint (2.12.2)

[error] func isDueTomorrowFilter is unused

(unused)